analysis/webservice/nexus_tornado/app_builders/NexusAppBuilder.py

```diff
@@ -31,7 +31,10 @@ def __init__(self):
 
         class VersionHandler(tornado.web.RequestHandler):
             def get(self):
-                self.write(pkg_resources.get_distribution("nexusanalysis").version)
+                analysis_version = pkg_resources.get_distribution("nexusanalysis").version
```

Comment:
Aren't these always the same? We are using a common version across both components (see setup.py/VERSION.txt)

Reply:
Currently they are, but could we possibly want them to be different versions at some point?

Reply:
I don't think so, we actually made this consolidation update somewhat recently

Reply:
I see. I'll roll this back then.

Reply:
Done